### PR TITLE
Move pip install command to a snippet

### PR DESCRIPTION
**Pull request by djsauble**
Duplicate code blocks are difficult to update because you must update all code blocks simultaneously. If you miss one, you risk breakage or inconsistency between otherwise identical examples.

To minimize risk, let's use [snippets](https://mintlify.com/docs/reusable-snippets) to deduplicate the code, so that you only need to update the snippet for the change to be reflected in all instances of the code.

### Example

Simplified example:

/snippets/installation.mdx
```
pip install prefect
```

Now you can import this code with:

```
import Installation from "/snippets/installation.mdx"

<Installation />
```

### Checklist

- [X] This pull request includes a label categorizing the change e.g. `maintenance`, `fix`, `feature`, `enhancement`, `docs`.
- [ ] ~This pull request references any related issue by including "closes `<link to issue>`"~
- [ ] ~If this pull request adds new functionality, it includes unit tests that cover the changes~
- [ ] ~If this pull request removes docs files, it includes redirect settings in `mint.json`.~
- [ ] ~If this pull request adds functions or classes, it includes helpful docstrings.~

### CODE REVIEW

**Review by billpalombi — APPROVED** [review]
lgtm

**Review by cicdw — APPROVED** [review]
Love this feature - thanks for wiring it up and showing how it's done!

File: docs/snippets/installation.mdx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Install the Prefect client.
+
+```bash
+pip install -U prefect==3.0.0rc3
```
Comment (discdiver) [inline docs/snippets/installation.mdx:4]:
```suggestion
pip install -U prefect --pre
```

File: docs/snippets/installation.mdx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Install the Prefect client.
```
Comment (discdiver) [inline docs/snippets/installation.mdx:1]:
```suggestion
Install Prefect.
```

This section is installing the client and server.

**Review by discdiver — COMMENTED** [review]
Thank you! Two suggestions.

**Pull request merged**